ruby-challenges.rb

```diff
@@ -32,3 +60,19 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+# declare the function, name it 'palindrome', it takes in a string as an argument.
+# for the  method, we add .downcase to the string so that it could be uniform whether the user typed capitalized or not. it would measure the same thing... a downcased verison.
+# we compare the string to see if it is strictly equal to the same string.reversed. 
+# if it does, then its a palindrome, if not then its not.
+def palindrome(string)
+    if string.downcase === string.downcase.reverse
+        p "#{string} is a palindrome" 
+    elsif p "#{string} is not a palindrome"
+    else p "something is brokeneth"
```

Comment:
Lol, nice!  I definitely look forward to looking for the fun additions you add to your assessments.  I really appreciate you making it fun 🎉